config/splits/vpge/config_split.patch.user.role.site_manager.yml

```diff
@@ -1,4 +1,8 @@
 adding:
   permissions:
     config_split_sequence_29i3per77hhj5b7j: 'assign custm_vpge_site_manager role'
+    config_split_sequence_1_29i3per77hhj5b7j: 'assign custm_vpge_site_manager role'
+    config_split_sequence_2_29i3per77hhj5b7j: 'assign custm_vpge_site_manager role'
+    config_split_sequence_3_29i3per77hhj5b7j: 'assign custm_vpge_site_manager role'
+    config_split_sequence_4_29i3per77hhj5b7j: 'assign custm_vpge_site_manager role'
```

Comment:
You can revert this. I don't know why it comes back every time. Just dedupe it.

Reply:
Done